.github/workflows/pull-request.yml

```diff
@@ -29,11 +29,11 @@ jobs:
       - name: Install packages
         run: yarn install --frozen-lockfile
 
+      - name: Run Build
+        run: yarn build
+
```

Comment:
Had to move this up because:
- `lib` folders for packages are in `.gitignore`
- `@bothrs/react-native-toast` depends on `@bothrs/react-native-layout`